From 9cfe5b505a8c770f8b31cb05cfc783f4ca5d0ccd Mon Sep 17 00:00:00 2001
From: Yannick Fertre <yannick.fertre@foss.st.com>
Date: Wed, 20 Sep 2023 12:57:58 +0200
Subject: [PATCH 0582/1141] arm64: dts: st: rework display support in
 stm32mp257f-ev1

Remove goodix & panel rm68200 support (board MB1230C).

Change-Id: I6bfb71400007d3f576cff5f66ac2c7e71d5e454e
Signed-off-by: Yannick Fertre <yannick.fertre@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/329410
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 arch/arm64/boot/dts/st/stm32mp257f-ev1.dts | 36 +---------------------
 1 file changed, 1 insertion(+), 35 deletions(-)

diff --git a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
index 1c25e958d46f..25df117cb703 100644
--- a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
@@ -200,31 +200,10 @@ dsi_in: endpoint {
 		port@1 {
 			reg = <1>;
 			dsi_out1: endpoint {
-				remote-endpoint = <&panel_in_dsi>;
-			};
-		};
-
-		port@2 {
-			reg = <2>;
-			dsi_out2: endpoint {
 				remote-endpoint = <&adv7535_in>;
 			};
 		};
 	};
-
-	panel_dsi: panel-dsi@0 {
-		compatible = "raydium,rm68200";
-		reg = <0>;
-		reset-gpios = <&gpiog 14 GPIO_ACTIVE_LOW>;
-		backlight = <&panel_dsi_backlight >;
-		status = "disabled";
-
-		port {
-			panel_in_dsi: endpoint {
-				remote-endpoint = <&dsi_out1>;
-			};
-		};
-	};
 };
 
 &eth1 {
@@ -356,7 +335,7 @@ ports {
 			port@0 {
 				reg = <0>;
 				adv7535_in: endpoint {
-					remote-endpoint = <&dsi_out2>;
+					remote-endpoint = <&dsi_out1>;
 				};
 			};
 
@@ -377,19 +356,6 @@ ili2511: ili2511@41 {
 		reset-gpios = <&gpiog 14 GPIO_ACTIVE_LOW>;
 		status = "okay";
 	};
-
-	gt9147: goodix-ts@5d {
-		compatible = "goodix,gt9147";
-		reg = <0x5d>;
-		pinctrl-names = "default";
-		pinctrl-0 = <&goodix_pins_a>;
-		panel = <&panel_dsi>;
-		interrupt-parent = <&gpioi>;
-		interrupts = <13 IRQ_TYPE_EDGE_FALLING>;
-		AVDD28-supply = <&scmi_v3v3>;
-		VDDIO-supply = <&scmi_v3v3>;
-		status = "disabled";
-	};
 };
 
 &i2s2 {
-- 
2.39.2

